lib/order.rb

```diff
@@ -0,0 +1,47 @@
+class Order
+
+  attr_reader :basket
+
+  def initialize
+    @basket = []
+  end
+
+  def add_to_basket(menu, dish_number, quantity)
+    quantity.times do
+      @basket << menu.select(dish_number)
+    end
+  end
+
+  def summary
+    fail "Basket empty" if empty_basket?
+    basket_hash = {}
+    basket.uniq.each do |a|
+      basket_hash[a] = basket.count(a)
+    end
+    basket_hash.each_pair do |k, v|
+      puts "#{v}x #{k.name} | £#{'%.2f' % (v * k.price)}"
```

Comment:
Style/FormatString: Favor `format` over `String#%`.<br>Rails/Output: Do not write to stdout. Use Rails' logger if you want to log.